sipbuild/module/source/12/siplib.c

```diff
@@ -7776,32 +7776,35 @@ static void sip_api_abstract_method(const char *classname, const char *method)
             classname, method);
 }
 
+
 /*
  * Report a deprecated class or method.
  */
 int sip_api_deprecated(const char *classname, const char *method)
 {
-  return sip_api_deprecated_12_16( classname, method, NULL );
+  return sip_api_deprecated_12_16(classname, method, NULL);
 }
 
+
 /*
- * Report a deprecated class or method with a given message.
+ * Report a deprecated class or method with an optional message.
  */
-int sip_api_deprecated_12_16(const char *classname, const char *method, const char *message)
+int sip_api_deprecated_12_16(const char *classname, const char *method,
+        const char *message)
 {
-    const unsigned int bufsize = 100 + ( message ? strlen(message) : 0 );
-    char buf[bufsize];
+    char buf[100];
```

Comment:
@philthompson10 This cause crashes when the total message (dynamic = static part) exceeds 100 characters.

I could propose this modification again but you write in your commit comment "Reverted changes to the `sip` module sources so that they build using MSVC", can you tell me what was not building using MSVC ? It looks like fine C code to me and the other modifications are only cosmetic.